### ISSUE TITLE
traccar web . the devices are not show

### ISSUE THREAD

**Issue by trasanda** [traccar/traccar#2045]
Hello,
I installed web traccar and thank Tananev has been successful, but now if I add the device (gv300) in traccar you see only once and then vanish when reloading the page but you see in mysql. Even if a group do you see the first time but then vanish.
![traccar](https://cloud.githubusercontent.com/assets/20000297/16302915/f6ba9676-394d-11e6-9d3a-ae6957893e45.PNG)
![mysql 2](https://cloud.githubusercontent.com/assets/20000297/16302917/f6beb29c-394d-11e6-8962-075594fbe663.PNG)

help?

**Comment by tananaev** [comment]
Have you added it via web interface? Check browser developer tools to see if there are any errors there.

**Comment by trasanda** [comment]
yes, i added via web interface. in the browser debugging it seems that there are no errors

**Comment by trasanda** [comment]
this log tracker-server

2016-06-23 14:39:18  INFO: [31CA6A0F] connected
2016-06-23 14:39:18 DEBUG: [31CA6A0F: 5004 < 5.90.34.249] HEX: 2b524553503a47544652492c3036303630332c3836323839343032323239313438312c2c2c31302c312c332c302e302c302c3331382e342c372e3537343434382c34352e3034313532302c32303136303632333132333734382c303232322c303031302c324232372c353241412c30302c302e302c2c2c2c3130302c3231303130302c2c2c2c32303136303632333132333735322c3346433524
2016-06-23 14:39:18  WARN: Unknown column 'device_id' in 'field list' - MySQLSyntaxErrorException (... < QueryBuilder:431 < DataManager:440 < DefaultDataHandler:27 < ...)
2016-06-23 14:39:18  INFO: [31CA6A0F] id: 4, time: 2016-06-23 14:37:48, lat: 45.04152, lon: 7.57445, speed: 0.0, course: 0.0
2016-06-23 14:39:38 DEBUG: [31CA6A0F: 5004 < 5.90.34.249] HEX: 2b524553503a47544652492c3036303630332c3836323839343032323239313438312c2c2c31302c312c332c302e302c302c3331382e342c372e3537343434382c34352e3034313532302c32303136303632333132333734382c303232322c303031302c324232372c353241412c30302c302e302c2c2c2c3130302c3231303130302c2c2c2c32303136303632333132333735322c3346433524
2016-06-23 14:39:39  WARN: Unknown column 'device_id' in 'field list' - MySQLSyntaxErrorException (... < QueryBuilder:431 < DataManager:440 < DefaultDataHandler:27 < ...)
2016-06-23 14:39:39  INFO: [31CA6A0F] id: 4, time: 2016-06-23 14:37:48, lat: 45.04152, lon: 7.57445, speed: 0.0, course: 0.0

**Comment by tananaev** [comment]
Are you using default version of Traccar?

**Comment by trasanda** [comment]
yes...
this
https://github.com/tananaev/traccar/releases/download/v3.5/traccar-windows-64-3.5.zip

**Comment by trasanda** [comment]
tips?

**Comment by trasanda** [comment]
anything?

**Comment by tananaev** [comment]
There is no need to spam. I won't answer you when I'm sleeping.

Are you using default config without any modifications?

**Comment by trasanda** [comment]
I'm sorry, I forgot you're from auckland and think that my son lives there (we are Italian). It is the conf is basic. I changed only user and password and suggested changes to traccar 3.5. Now there is also another problem. since I changed username and password, the address does not open anymore. traccar the log says it is waiting. if you want tomorrow I can give you access, there are business secrets.
